cypress/support/commands.js

```diff
@@ -20,7 +20,7 @@ Cypress.Commands.add('idapiMockNext', (status, body) => {
   cy.request(getMockOptions(status, body));
 });
 
-Cypress.Commands.add('idapiMockAll', (status, body, path) => {
+Cypress.Commands.add('mockAll', (status, body, path) => {
```

Comment:
Are we using this command, maybe we can remove it?

Reply:
also being used in the onboarding flow spec (may need to expand the diff in github)

Reply:
Ah! Gotcha